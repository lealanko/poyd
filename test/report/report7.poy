(* Build report with treestats *)
read ("1.fas", "30.ss" )
set(seed:50)
build (1)
report (treestats)