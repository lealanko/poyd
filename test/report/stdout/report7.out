Trees Found:
                                            
              Tree length    Number of hits
              3253.          1
